packages/account/src/Sections/Profile/PhoneVerification/resend-code-timer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef } from 'react';
+import { Button, CaptionText } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+
+type TResendCodeTimer = {
+    resend_code_text: string;
+};
+const ResendCodeTimer = ({ resend_code_text }: TResendCodeTimer) => {
+    // TODO: Use dynamic value for setting initial value for counter once mockApi for timestamp is finalised
+    const [timer, setTimer] = React.useState(60);
+    const [start_timer, setStartTimer] = React.useState(false);
+    const request_code = useRef(resend_code_text);
+
+    React.useEffect(() => {
+        let countdown: NodeJS.Timeout;
```

Comment:
```suggestion
        let countdown: ReturnType<typeof setInterval>;
```

Reply:
May I ask what is the difference here 😄

Reply:
here what gemini says:
`In a React component, both NodeJS.Timeout and ReturnType<typeof setInterval> can work, but ReturnType<typeof setInterval> is generally considered more correct for the following reasons:

Platform Independence: NodeJS.Timeout is specific to Node.js, while ReturnType<typeof setInterval> works in both browsers and Node.js environments. This is important if your React component might be used in both server-side rendering (SSR) and client-side rendering.
Type Safety: ReturnType<typeof setInterval> captures the exact return type of setInterval based on your environment (number in browsers, potentially NodeJS.Timeout in Node.js). This provides better type safety and helps prevent potential errors.
Here's a breakdown of the scenarios:

Pure Client-Side React Component: If your component only runs in the browser, both options would work. However, ReturnType<typeof setInterval> still offers better type safety.
React Component with SSR: If your component might be rendered on the server (Node.js) before being hydrated in the browser, NodeJS.Timeout could cause type errors. ReturnType<typeof setInterval> will handle both environments gracefully.
Recommendation:

For most React components, it's recommended to use ReturnType<typeof setInterval> to ensure type safety and platform independence. This approach provides a more robust and flexible solution for your React development.`

Reply:
Wow! thanks :)